app/controllers/api/v1/data_studio/school_rollout_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module DataStudio
+      class SchoolRolloutController < Api::ApiController
```

Comment:
I think we need to stop lead provider tokens from working with this endpoint. This includes details of schools who have opted out, which we don't want to share with them. I'm also not sure we're allowed to share which providers have recruited which schools, which we will be adding very soon

Reply:
Yes, was going to add the private endpoint code that @asmega wrote